code/implementation/system-stack/src/main/java/io/cattle/platform/systemstack/listener/external/api/catalog/Template.java

```diff
@@ -0,0 +1,376 @@
+package io.cattle.platform.systemstack.listener.external.api.catalog;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
In general avoid jackson annotations if possible.  Here, none of them are required, so just remove all the `@JsonProperty`s